examples/ui-demo/src/app/api/rpc-base-sepolia/route.ts

```diff
@@ -3,21 +3,30 @@ import { NextRequest, NextResponse } from "next/server";
 import { env } from "../../../../env.mjs";
 
 export async function POST(req: NextRequest) {
-  const body = await req.text();
+  const body = await req.json();
```

Comment:
updating this handler to match the other rpc proxy

Reply:
i just copied this from the other handler. i'm pretty sure it's just b/c we include the api keys in our environment variables for the rpc base urls, but not the `ALCHEMY_API_URL`.